ge/org.osate.ge.errormodel/src/org/osate/ge/errormodel/ui/swt/TypeTokenListEditorModel.java

```diff
@@ -70,4 +70,8 @@
 	String getTypeTokenLabel(TypeToken value);
 
 	String getTypeTokensLabel();
+
+	default <T> Stream<T> getSelectedElements(final Class<T> clazz) {
```

Comment:
I'm not sure what this is.  This introduces the concept of a selection into the model. I'm not sure if that's a good idea or what it means. If it is something that is needed in the model, it does not seem likely that it should have a default implementation.